l10n_co_geo/data/res_country_state.xml

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I investigated on one of geonames sources (http://geonames.nga.mil/namesgaz/) which is also very interesting, I'll get in contact with them and let's see what we can do. This definitely can be a good thing.